ssl/ssl_ciph.c

```diff
@@ -1311,7 +1311,8 @@ static int ciphersuite_cb(const char *elem, int len, void *arg)
     cipher = ssl3_get_cipher_by_std_name(name);
     if (cipher == NULL) {
         SSLerr(SSL_F_CIPHERSUITE_CB, SSL_R_NO_CIPHER_MATCH);
```

Comment:
Setting this error here no longer seems appropriate. Since we're going to continue to process the rest of the list we might get a subsequent cipher match. Probably we should delay raising this error until we notice that the returned stack has no elements in it in `set_ciphersuites`.

Reply:
Yes, error there is no longer appropriate. I've moved this error into function set_ciphersuites().

I was originally thinking about some "warnings", so user should know, that some of entered ciphersuites can't be recognized., that was reason why I keeped this error.